hack/prebuild/local-dev/README.md

```diff
@@ -0,0 +1,37 @@
+# Setting up a local libvirt dev environment
+
+## Set up NetworkManager DNS overlay
+_This step is optional, but useful for being able to resolve cluster-internal hostnames from your host._
+
+Edit `/etc/NetworkManager/NetworkManager.conf` and set `dns=dnsmasq` in section `[main]`.
+Tell dnsmasq to use your cluster. The syntax is `server=/<baseDomain>/<firstIP>`.
+
+For this example:
+```
+> echo server=/tt.testing/192.168.124.1 | sudo tee /etc/NetworkManager/dnsmasq.d/tectonic.conf
+> systemctl restart NetworkManager
+```
+## Install the terraform provider
+Make sure you have the virsh binary installed: `sudo dnf install libvirt-client libvirt-devel`
+Install the libvirt terraform provider:
+```
+GOBIN=~/.terraform.d/plugins go get github.com/crawford/terraform-provider-libvirt
```

Comment:
why is this pointing to a fork? could we just install with tf init?

Reply:
@enxebre at the moment, terraform can only automatically install plugins officially distributed by hashicorp, so we need to manually install it: https://www.terraform.io/docs/configuration/providers.html#third-party-plugins